src/main/java/org/broadinstitute/hellbender/tools/spark/sv/utils/FileUtils.java

```diff
@@ -20,4 +23,36 @@ public static void writeLinesToSingleFile(final Iterator<String> linesToWrite, f
             throw new GATKException("Can't write "+fileName, ioe);
         }
     }
+
+    public static void writeSAMFile(final Iterator<SAMRecord> alignments, final SAMFileHeader header, final String outputName,
+                                    final boolean preOrdered) {
+        try (final SAMFileWriter writer = ReadUtils.createCommonSAMWriter(new File(outputName), null, header,
+                preOrdered, true, false) ) {
+            alignments.forEachRemaining(writer::addAlignment);
+        } catch ( final UncheckedIOException ie) {
+            throw new GATKException("Can't write SAM file to the specified location: " + outputName, ie);
+        }
+    }
+
+    public static boolean createDirInBucketToWriteTo(final String pathString) {
```

Comment:
This method isn't really bucket-specific so it probably doesn't need `bucket` in the name.